docs/sources/mimir/release-notes/v2.10.md

```diff
@@ -21,24 +21,24 @@ The highlights that follow include the top features, enhancements, and bugfixes
 - **Improved the performance of the OTLP ingestion** and more detailed information was added to the traces in order to make troubleshooting problems easier.
 - **Improved the performance of series matching in the store-gateway** by always including the `__name__` posting group causing a reduction in the number of object storage API calls.
 - **Improved the performance of _label values with matchers_** calls when number of matched series is small. If you're using Grafana to query Grafana Mimir make sure you configure the right version in your Prometheus datasource configuration.
```

Comment:
What would the "right version" be? Can we just give a version number?

Reply:
I think this is the relevant PR:
https://github.com/grafana/mimir/pull/5600

Reply:
I didn't want to reference the specific value of that configuration because it might change over time and get outdated here. Right the correct version there would be: `Mimir >2.3.0`, but who knows if in a week we release a datasource version which will have a `Mimir 2.10.0` option in that field.

Reply:
> Right the correct version there would be: Mimir >2.3.0, but who knows if in a week we release a datasource version which will have a Mimir 2.10.0 option in that field.

I'm confused... If we write "Mimir >2.3.0", isn't "Mimir 2.10.0" greater than 2.3, which would mean that making the statement >2.3.0 is perfectly fine? 

Maybe we could say 

If you're using Grafana to query Grafana Mimir, you'll need to be sure your [Prometheus data source configuration](https://grafana.com/docs/grafana/latest/datasources/prometheus/configure-prometheus-data-source/#performance) has the `Prometheus type` set to `Mimir` and the `Version` set correctly in order to benefit from this improvement.

Reply:
@colega let me know what you think!